server/src/main/java/com/testsigma/service/TestStepService.java

```diff
@@ -653,6 +653,13 @@ public void deleteStepsByTestCaseId(Long testCaseId){
     public Optional<TestStep> findTopByTestCaseIdOrderByPositionDesc(Long testCaseId) {
         return this.repository.findTopByTestCaseIdOrderByPositionDesc(testCaseId);
     }
+    public List<TestStep> findAllByStepGroupId(Long stepGroupId){
```

Comment:
@nixonakash01 Where is this function used ?

Reply:
@tarun-testsigma its not being used, ill remove it and commit again